modules/core/src/test/scala/com.snowplowanalytics.snowplow.bigquery/processing/ProcessingSpec.scala

```diff
@@ -153,6 +157,141 @@ class ProcessingSpec extends Specification with CatsEffect {
     }
   }
 
+  def e4_2 = {
+    val data = json"""{ "myString": "my value"}"""
+    val contexts = Contexts(
+      List(
+        SelfDescribingData(
+          SchemaKey.fromUri("iglu:com.snowplowanalytics.snowplow.media/ad_click_event/jsonschema/1-0-0").toOption.get,
+          data
+        )
+      )
+    )
+
+    val mocks = Mocks.default.copy(
+      addColumnsResponse = MockEnvironment.Response.Success(
+        FieldList.of(
+          BQField.of(
+            "contexts_com_snowplowanalytics_snowplow_media_ad_click_event_1",
+            StandardSQLTypeName.STRUCT,
```

Comment:
If this is for a context, then I do not know if this should be `STRUCT` or `ARRAY`.

Reply:
I think it's still `STRUCT` but with `REPEATED` mode